core/pom.xml

```diff
@@ -100,7 +100,7 @@
         <jakarta.enterprise.deploy-api.version>1.7.2</jakarta.enterprise.deploy-api.version>
         <jakarta.resource-api.version>2.1.0</jakarta.resource-api.version>
         <jakarta.annotation-api.version>3.0.0</jakarta.annotation-api.version>
-        <exousia.version>2.1.2</exousia.version>
+        <exousia.version>3.0.0-M3</exousia.version>
```

Comment:
I'd be careful about blleding PRs together - this upgrade breaks quicklook.
Might be worth doing Exousia in a separate PR